src/Instance/InstanceMovement/InstanceMovement.js

```diff
@@ -1,82 +1,41 @@
-import React, {
-  useCallback,
-} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
 import {
   Paneset,
 } from '@folio/stripes/components';
 
-import {
-  InstanceDetails,
-} from '../InstanceDetails';
-import {
-  HoldingsListContainer,
-} from '../HoldingsList';
-import {
-  MoveItemsContext,
-} from '../MoveItemsContext';
+import { InstanceMovementDetailsContainer } from './InstanceMovementDetails';
 
 const InstanceMovement = ({
-  referenceData,
   instanceFrom,
   instanceTo,
   onClose,
 }) => {
-  const closeInstanceFrom = useCallback(() => {
-    onClose(instanceFrom);
-  }, [instanceFrom, onClose]);
-
-  const closeInstanceTo = useCallback(() => {
-    onClose(instanceTo);
-  }, [instanceTo, onClose]);
-
   return (
     <Paneset data-test-movement>
-      <InstanceDetails
+      <InstanceMovementDetailsContainer
         instance={instanceFrom}
-        referenceData={referenceData}
-        onClose={closeInstanceFrom}
+        onClose={onClose}
         data-test-movement-from-instance-details
-      >
-        <MoveItemsContext>
-          <HoldingsListContainer
-            instance={instanceFrom}
-            referenceData={referenceData}
-            draggable={false}
-            droppable={false}
-          />
-        </MoveItemsContext>
-      </InstanceDetails>
+      />
 
-      <InstanceDetails
+      <InstanceMovementDetailsContainer
```

Comment:
I think having defaults 
```
  instanceFrom: {},
  instanceTo: {},
```
 for the `<InstanceMovement />` component is not possible as it would lead to errors if the component would be instantiated with them. I suggest removing these defaults and mark them as required.

Reply:
Actually, it would be nice to have this as a rule of thumb - to not have default values as empty objects or arrays

Reply:
I'm pretty sure it wouldn't lead to errors when these props are not passed

Reply:
It's a good thing that you're sure, but I can refer to MCL, that makes thing difficult if you pass default empty array before your data is loaded - cell size measure will not work correctly.

Reply:
So try it and we will see if you are sure.

Reply:
<img width="785" alt="Screen Shot 2020-07-02 at 21 29 47" src="https://user-images.githubusercontent.com/40821852/86396792-4b7af400-bcab-11ea-97e2-ef72ba78d620.png">
<img width="562" alt="Screen Shot 2020-07-02 at 21 29 12" src="https://user-images.githubusercontent.com/40821852/86396797-4e75e480-bcab-11ea-97b9-84bb0b4d02e0.png">

Reply:
ah, this error is thrown not by `InstanceMovementDetailsContainer`

> Anyways, what is the purpose of allowing to render the content if it is an empty object? It would lead to displaying of the content which is not quite ready or would possibly end up in error. I tracked down that everything is expected to work with the empty object which does not really make sense.

e.g. there is an error with data loading

options here:
for all show toast with "Smth is wrong"
1. to show nothing, empty pane, and with it props could be required
2. to show `the content which is not quite ready`
3. or actually show an error in the pane

@aliaksei-chumakou it's not the case with MCL, previously we discussed how to display pane when smth is wrong - content should be visible to end user # 2, but I agree about MCL (and IMO it should do recalculations when contentData is changed, but it's another story)

@VictorSoroka1 thanks for pointing possible error - it's holding loading

Reply:
Options that you listed I think is worth asking the PO. In data-export I prefer to show error notifications, but we don't have so many endpoints there. Also, I see that many apps are using the in build error alerts (e.g. `throwErrors: true`) which does not bring good UI experience but at least states that the error occurs. Don't have a strong opinion on that. My initial comment was about the possible errors because of the default props.